common/element_handle.go

```diff
@@ -263,11 +268,7 @@ func (h *ElementHandle) fill(_ context.Context, value string) error {
 	if err != nil {
 		return err
 	}
-	r, ok := result.(goja.Value)
-	if !ok {
-		return fmt.Errorf("expected goja value; got %T", result)
-	}
-	if s := r.String(); s != resultDone {
+	if s := gojaValueToString(h.ctx, result); s != resultDone {
```

Comment:
Are we changing the behaviour here? It used to return an error, but now panics

Reply:
Ah, nice catch! You might be right, I'll check.

Reply:
I sent the updates, please let me know 🙇